### PR TITLE
Update destination file if default when updating source file

### LINKED ISSUE #2: Destination file name does not change when changing Source file
When selecting a new Source file after encrypting a file, the destination file name does not change to follow source.
Desired behavior: destination file follows source.

### PR DESCRIPTION
**Pull request by mjarosie**
Fixes #2. I fixed it as I couldn't be bothered restarting the application with each file, nor manually renaming the destination with each pdf file. Hopefully that's a desired behaviour? I've decided not to overwrite the destination path when user specified it manually beforehand.

Known bug: If the output file text contains rubbish text which doesn't point to any file, the application would crash when calling `System.IO.Path.GetDirectoryName` to append the suffix. This PR doesn't change this behaviour.

### CODE REVIEW

**Comment by ryangriggs** [comment]
As you noted, if the user chooses an output filename (or types it manually) which doesn't match the standard format ([filename]-encrypted.pdf), then it won't overwrite the output filename for the newly selected input file.  This could lead to confusion about **why** it didn't overwrite the output filename, as this overwrite behavior would be expected in most cases.  I think it would be more consistent to **always** overwrite the output filename when selecting an input file. Then if the user needs to change the output destination, they can manually select the desired output file after selecting the input.

File: frmMain.cs [hunk 1]
```diff
@@ -32,19 +32,24 @@ public frmMain()
...
+
 			DialogResult result = dlgOpen.ShowDialog();
 			if (result == DialogResult.Cancel) { return; }
 
 			txtInputFile.Text = dlgOpen.FileName;
 
-			// Generate output filename matching input with "-encrypted" if none is specified.
-			if (txtOutputFile.Text == "")
+			// Generate output filename matching input with "-encrypted" if none is specified or if overwriting the one generated by default.
+			if (txtOutputFile.Text == "" || txtOutputFile.Text == GetFilenameWithSuffix(previouslySelectedInputFile))
```
Comment (just-ajs) [inline frmMain.cs]:
```suggestion
			if (String.IsNullOrEmpty(txtOutputFile.Text) || txtOutputFile.Text == GetFilenameWithSuffix(previouslySelectedInputFile))
```

File: frmMain.cs [hunk 2]
```diff
@@ -32,19 +32,24 @@ public frmMain()
 			InitializeComponent();
 		}
 
+		private string GetFilenameWithSuffix(string file, string suffix = "-encrypted")
+		{
+			return System.IO.Path.GetDirectoryName(file) + "\\" + System.IO.Path.GetFileNameWithoutExtension(file) + suffix + ".pdf";
```
Comment (just-ajs) [inline frmMain.cs]:
```suggestion
			return $"{System.IO.Path.GetDirectoryName(file)}\\{ System.IO.Path.GetFileNameWithoutExtension(file)}{suffix}.pdf";
```

File: frmMain.cs (hunk 1 above)
Comment (mjarosie) [reply frmMain.cs]: I've gotten rid of the check completely now 👍

File: frmMain.cs (hunk 2 above)
Comment (mjarosie) [reply frmMain.cs]: Good spot, thanks! I've used `System.IO.Path.Combine` as well.

**Comment by mjarosie** [comment]
All comments addressed.

**Pull request merged**